files/zh-cn/web/html/attributes/crossorigin/index.html

```diff
@@ -78,5 +78,5 @@ <h3>link crossorigin</h3>
 <h2 id="另请参阅">另请参阅</h2>
 
 <ul>
- <li><a href="/zh-CN/docs/Web/HTTP/Access_control_CORS" title="en/HTTP access control">HTTP access control</a>（HTTP 访问控制）</li>
+ <li><a href="/zh-CN/docs/Web/HTTP/Access_control_CORS" >HTTP access control</a>（HTTP 访问控制）</li>
```

Comment:
trailing space shoud be removed:

```suggestion
 <li><a href="/zh-CN/docs/Web/HTTP/Access_control_CORS">HTTP access control</a>（HTTP 访问控制）</li>
```

It's better to parse the `<a>` tag.

Reply:
you can refer this:

```go
func parseAnchor(data string) (href, content string, err error) {
	index := strings.Index(data, "href")
	if index == -1 {
		return "", "", fmt.Errorf("cannot find href")
	}
	data = data[index+len(href):]
	endIndex := strings.Index(data, "</a>")
	if endIndex == -1 {
		return "", "", fmt.Errorf("cannot find </a>")
	}
	token := rune(0)
	start, end := 0, 0
	for i, b := range data {
		switch b {
		case '"', '\'':
			if token == 0 {
				token = b
				if start == 0 {
					start = i
				}
			} else if token == b {
				if end == 0 {
					end = i
				}
				token = 0
			}
		case '>':
			if end != 0 && token == 0 {
				href = data[start+1 : end]
				content = data[i+1 : endIndex]
				return
			}
		}
	}
	return "", "", fmt.Errorf("cannot find href")
}
```

Reply:
Wow, good recommendation

Reply:
We only need content of the href in the a tag?

Reply:
yes

Reply:
oh, wait a minute